api/controllers/site.js

```diff
@@ -124,26 +127,23 @@ module.exports = {
 
       return authorizer.removeUser(req.user, site);
     })
-    .then(() =>
-      SiteMembershipCreator.revokeSiteMembership({ user: req.user, site, userId })
-    )
-    .then(() =>
-      UserActionCreator.addRemoveAction({
+      .then(() => SiteMembershipCreator
```

Comment:
is this a linting issue? 100 characters?

Reply:
Lint error. No line break after arrow function